validate_data.py

```diff
@@ -86,6 +87,8 @@ def prune_nodata_list(data):
             pruned_value = prune_nodata(value)
             if len(pruned_value) > 0:
                 pruned.append(pruned_value)
+        else:
```

Comment:
Little worried about this, would this be adding a kind of wildcard to what is accepted --?  (Kinda hard for me to read/understand in context tho!)

Reply:
Your Spidey-sense was good on this Bruce! This had been for a special case in which we returned the flat list from Rasdaman which has since been removed! Thanks for catching this!